extension/persistence/eclipselink/build.gradle

```diff
@@ -17,9 +17,10 @@
 dependencies {
     implementation(project(":polaris-core"))
     implementation(project(":polaris-service"))
-    implementation("org.eclipse.persistence:eclipselink:4.0.3")
-    implementation("io.dropwizard:dropwizard-jackson:${dropwizardVersion}")
+    implementation(libs.eclipselink)
+    implementation(platform(libs.dropwizard.bom))
+    implementation("io.dropwizard:dropwizard-jackson")
```

Comment:
Is there a reason this one doesn't have a "libs." identifier?

Reply:
Yup - it's just some artifact that's "version managed" via a bom.